FrontEnd/myapp/src/Stock.js

```diff
@@ -66,8 +78,19 @@ class Stock extends React.Component {
               marker: { color: "red" },
             },
           ]}
-          layout={{ width: 720, height: 440, title: StockSymbol }}
+          layout={{ width: 780, height: 440 }}
         />
+        <Container>
+          <Header as="h2" style={{ textAlign: "center", margin: 20 }}>
+            News about {this.state.ticker}
+          </Header>
+          {this.state.articles.length > 0 && (
+            <ArticleList articles={this.state.articles} />
+          )}
+          {this.state.apiError && (
+            <p>Could not fetch any articles. Please try again.</p>
+          )}
+        </Container>
```

Comment:
Smart to make sure this is updated with the other page

Reply:
I believe it will be good to have them synced together as so